deployment/stack.py

```diff
@@ -147,10 +146,11 @@ def make_template(config, config_yaml):
         )
 
     volume_properties = {
-        'AvailabilityZone': config['AVAILABILITY_ZONE'],
         'Encrypted': True,
         'Size': config['DATA_VOLUME_SIZE'],
         'Tags': load_tags(),
+        'AvailabilityZone': functions.get_att(
+            'WebInstance', 'AvailabilityZone'),
```

Comment:
A nitpick: if an open parenthesis is on a line by itself then the closing one should be as well